test/unit/bootstrap.ts

```diff
@@ -2,18 +2,18 @@
 /// <reference path="../../src/types.d.ts" />
 
 import 'dotenv/config'
-import RedisStore from '../../src/RedisStore'
-import { config } from '../../src/config'
+import RedisStore from '../../src/RedisStore.js'
```

Comment:
We write typescript, so why `.js` and not `.ts`?

Reply:
Withoud `.js` extension I have this ts-lint error in my IDE: 
`Relative import paths need explicit file extensions in EcmaScript imports when '--moduleResolution' is 'node16' or 'nodenext'. Consider adding an extension to the import path.`

So I have to add `.js` at the end of import to fix this problem. By the way, this approach is used in a lot of places on the mwoffliner. More info about this bug can be found here ( https://github.com/microsoft/TypeScript/issues/49083 ) but I don’t think we need to pay much attention to it since it doesn’t break anything.

Reply:
@VadimKovalenkoSNF Please open a dedicated ticket about that (I don't really have the time right now to go through the whole stuff). Even if this looks like a detail, I hardly can understand how it could be OK to have Typescript code in `.js`.

Reply:
I don't think it does make sense to open a separate ticket since this is Typescript side effect and not a mwoffliner bug.

Reply:
where the root cause is located, is secondary. What is portant is impact and tracking of it. Without ticket, we can not track.

Reply:
I opened related ticket - https://github.com/openzim/mwoffliner/issues/1900